src/context/items.js

```diff
@@ -0,0 +1,19 @@
+import React, { createContext } from 'react'
+
+export default ItemProvider
+
+export const itemContext = createContext({
+    item: '',
+    updateItem: () => null
+})
+
+const ItemProvider = ({children}) => {
+    const [item, setItem] = useState ('')
+
+    return(
+        <itemContext.provider value={{item, setItem: updateItem}}>
```

Comment:
setItem and updateItem are backwards here

Reply:
also swapped these!